deployment/localstack/terraform/main.tf

```diff
@@ -1,4 +1,11 @@
 module "aws" {
-  source              = "../../../iac/terraform/aws"
-  environment         = "localstack"
+  source         = "../../../iac/terraform/aws"
+  environment    = "localstack"
+  hosted_zone_id = aws_route53_zone.my_zone.zone_id
+}
+
+resource "aws_route53_zone" "my_zone" {
+  #checkov:skip=CKV2_AWS_38: "Ensure Domain Name System (DNS) query logging is enabled for Amazon Route 53 hosted zones"
+  #checkov:skip=CKV2_AWS_39: "Ensure Domain Name System Security Extensions (DNSSEC) signing is enabled for Amazon Route 53 public hosted zones"
```

Comment:
Why DNSSEC signing is skipped?

Reply:
Created another new ticket https://github.com/CraftsmenLtd/BloodConnect/issues/108 related to this problem.